CommunicateHealth/outlink.js
# Consider links to domains that contain the current domain name as outlinks

Links to maternalhealth.gov and womenshealth.gov were not considered "outlinks" from health.gov.  This code now looks for `"//"+domain_name` or `"///www." + domain_name` when determining if the href of a link points to an external site.

Task: https://communicatehlth.teamwork.com/#/tasks/32740969

To test, first:

```
git fetch
git checkout feature/32740969-fix-domain-exclusion
cp dist/outlink.js ../healthgov-refresh/docroot/node_modules/@communicatehealth/outlink.js/dist/
cd ../healthgov-refresh/docroot
fin start
fin frontend hg-js
fin drush cr
```
Then:

- Go to http://healthgov.docksal/console/login and log in 
- Go to http://healthgov.docksal/news/202012/usda-and-hhs-just-released-dietary-guidelines-americans-2020 and click Edit
- Change _Read the Dietary Guidelines_ link to `https://healthgov.docksal/`
- Change _Dietary Guidelines for Americans, 2020-2025_ link to `https://www.healthgov.docksal/`
- Change _Executive Summary_ link to `https://dietaryhealthgov.docksal/`
- Click on "Save"
- Confirm _Read the Dietary Guidelines_ link is not an outlink
- Confirm _Dietary Guidelines for Americans, 2020-2025_ link is not an outlink
- Confirm _Executive Summary_ link is an outlink
- Revert the revision if you wish
